tethys_apps/context_processors.py

```diff
@@ -41,4 +50,31 @@ def tethys_apps_context(request):
             if hasattr(app, 'enable_feedback'):
                 context['tethys_app']['enable_feedback'] = app.enable_feedback
 
+        # Web-Analytics services checks/Adding appropriate tags
+        context['analytics'] = {
+            'analytical': bool('analytical' in settings.INSTALLED_APPS),
+            'clickmap': bool(settings.CLICKMAP_TRACKER_ID),
```

Comment:
@rileyhales These checks all assume these are in settings, but the very well may not be (if someone updates without regenerating their template for example). Change these to `bool(settings.get('CLICKMAP_TRACKER_ID')`.